test/integration/autokey-simple/autokey_simple_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2022 Google LLC
```

Comment:
plz check all copyright dates for anything new.

Reply:
Good catch